packages/component/src/loadableReady.js

```diff
@@ -49,8 +49,11 @@ export default function loadableReady(
   let resolved = false
 
   return new Promise(resolve => {
-    window.__LOADABLE_LOADED_CHUNKS__ = window.__LOADABLE_LOADED_CHUNKS__ || []
-    const loadedChunks = window.__LOADABLE_LOADED_CHUNKS__
+    const prefix = namespace ? `__${namespace}` : ''
+    const loadedChunksKey = `${prefix}__LOADABLE_LOADED_CHUNKS__`
```

Comment:
Possibly a breaking change since `namespace` is separately used for `getRequiredChunkKey` above. Maybe I should add a separate option.